receiver/kafkareceiver/config.go

```diff
@@ -37,6 +37,11 @@ type HeaderExtraction struct {
 	Headers        []string `mapstructure:"headers"`
 }
 
+type Avro struct {
+	// AVRO schema used to decode messages
+	Schema string `mapstructure:"schema"`
```

Comment:
Is that schema a reference to a file path?

Reply:
It looks like it's a complex JSON object encoded as string, is that correct?

Reply:
yes, correct. Initially it was a file path but as I remember, @MovieStoreGuy prefered to have it inlined so I changed it.